File: csharp/fizzbuzz.cs
using System;

public class Test
{
  public static void Main()
  {
    for(var i = 0; i < 100; i++) {
     var msg = "";
     if(i % 3 == 0){
       msg += "fizz";
     } 
     if(i % 5 == 0){
       msg += "buzz";
     }
     if(msg == ""){
       Console.WriteLine(i);
       continue;
     }
     Console.WriteLine(msg);
    }
  }
}  

File: go/fizzbuzz.go
package main

import "fmt"

func main(){
	for i := 0; i < 100; i++ {
		msg := ""
		found := 0
		if i % 3 == 0 {
			msg += "fizz"
			found++
		}
		if i % 5 == 0 {
			msg += "buzz"
			found++
		}
		if found == 0 {
			fmt.Println(i)
		} else {
			fmt.Println(msg)
		}
	}
}
